mxd/3.1_MXD_tutorial1_dataexchange.md

```diff
@@ -0,0 +1,181 @@
+# Provide and consume data
+
+As described in the introduction, a data exchange between Bob (Data Provider) and Alice (Data Consumer) is to be tested.
+
+To maximize the benefit from this tutorial it is recommended to follow the tutorial in the given order.
+
+## Provide data
+
+In this step we will focus on inserting data into our participant Alice using
+the [Management API](https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api). We will use plain
+CLI tools (`curl`) for this, but feel free to use graphical tools such as Postman or Insomnia.
+
+Alice, as a data consumer, wants to consume data from Bob. Bob, as a data provider, needs to create an asset for Alice. The data asset should have the following properties:
+
+| ID          | 3                                                                                            |
+|-------------|----------------------------------------------------------------------------------------------|
```

Comment:
this table is spurious, and it confuses more than it contributes, as the data shown is also contained in the JSON example further down.